actors/miner/tests/util.rs

```diff
@@ -204,8 +261,1024 @@ impl ActorHarness {
+
+                    if pc_on_chain.info.replace_capacity {
+                        let replaced = self.get_sector(rt, pc_on_chain.info.replace_sector_number);
+                        if replaced.initial_pledge > pledge {
+                            pledge = replaced.initial_pledge;
+                        }
+                    }
+
+                    expected_pledge += pledge;
+                }
+            }
+
+            if expected_pledge != TokenAmount::from(0) {
+                rt.expect_send(
+                    *STORAGE_POWER_ACTOR_ADDR,
+                    PowerMethod::UpdatePledgeTotal as u64,
+                    RawBytes::serialize(BigIntSer(&expected_pledge)).unwrap(),
+                    TokenAmount::from(0),
+                    RawBytes::default(),
+                    ExitCode::Ok,
+                );
+            }
+        }
+    }
+
+    fn get_sector(&self, rt: &MockRuntime, sector_number: SectorNumber) -> SectorOnChainInfo {
+        let state = self.get_state(rt);
+        state.get_sector(&rt.store, sector_number).unwrap().unwrap()
+    }
+
+    fn advance_to_epoch_with_cron(&self, rt: &mut MockRuntime, epoch: ChainEpoch) {
+        let mut deadline = self.get_deadline_info(rt);
+        while deadline.last() < epoch {
+            self.advance_deadline(rt, CronConfig::empty());
+            deadline = self.get_deadline_info(rt);
+        }
+        rt.epoch = epoch;
+    }
+
+    pub fn advance_to_deadline(&self, rt: &mut MockRuntime, dlidx: u64) -> DeadlineInfo {
+        let mut dlinfo = self.deadline(rt);
+        while dlinfo.index != dlidx {
+            dlinfo = self.advance_deadline(rt, CronConfig::empty());
+        }
+        dlinfo
+    }
+
+    fn deadline(&self, rt: &MockRuntime) -> DeadlineInfo {
+        let state = self.get_state(rt);
+        state.recorded_deadline_info(&rt.policy, rt.epoch)
+    }
+
+    pub fn advance_deadline(&self, rt: &mut MockRuntime, mut cfg: CronConfig) -> DeadlineInfo {
+        let state = self.get_state(rt);
+        let deadline = new_deadline_info_from_offset_and_epoch(
+            &rt.policy,
+            state.proving_period_start,
+            rt.epoch,
+        );
+
+        if state.deadline_cron_active {
+            rt.epoch = deadline.last();
+            cfg.expected_enrollment = deadline.last() + rt.policy.wpost_challenge_window;
+            self.on_deadline_cron(rt, cfg);
+        }
+        rt.epoch = deadline.next_open();
+
+        let state = self.get_state(rt);
+        state.deadline_info(&rt.policy, rt.epoch)
+    }
+
+    fn on_deadline_cron(&self, rt: &mut MockRuntime, cfg: CronConfig) {
+        let state = self.get_state(rt);
+        rt.expect_validate_caller_addr(vec![*STORAGE_POWER_ACTOR_ADDR]);
+
+        // preamble
+        let mut power_delta = PowerPair::zero();
+        match cfg.detected_faults_power_delta {
+            Some(pwr) => {
+                power_delta += &pwr;
+            }
+            None => (),
```

Comment:
Wild rust idea feel free to ignore: what if you made power_delta evaluate toe the `match` statement.  so 
```rust
let out power_delta = match cfg.detected_faults_power_delta {
    Some(pwr) => {&pwr}
    None => {PowerPair::zero()}
};
```

I don't even know if this works and maybe yours is more idiomatic but evaluating match to unit feels funky.

Reply:
I think you just want `cfg.detected_faults_power_delta.unwrap_or(PowerPair::zero())`

Reply:
ah nice, i like the unwrap_or incantation.

Reply:
done, used the `unwrap_or` spell.